tests/integration-tests/conftest.py

```diff
@@ -968,11 +968,16 @@ def register_cli_credentials(initialize_cli_creds):
             register_cli_credentials_for_region(region, creds)
 
 
-@pytest.fixture(scope="session", autouse=True)
-def vpc_stacks(cfn_stacks_factory, request):
-    """Create VPC used by integ tests in all configured regions."""
+@xdist_session_fixture(autouse=True)
+def vpc_stacks_shared(cfn_stacks_factory, request, key_name):
```

Comment:
isn't it a problem to use the session scoped `cfn_stacks_factory` fixture in here?

Reply:
it should not, since cfn_stacks_factory has the same scope.
This isn't true for vpc_stack_with_endpoints, for which I'm going to make a change